src/pages/api/users.ts

```diff
@@ -0,0 +1,7 @@
+import { NextApiResponse } from 'next';
+import { prisma } from '../../../utils/prisma';
+
+export default async function handler(req, res: NextApiResponse) {
+  const users = await prisma.user.findMany();
```

Comment:
Probably should have a try catch here

Reply:
Might want to throw a 500 if there's an error.

Reply:
Otherwise this can occur:

```
 ⨯ PrismaClientKnownRequestError: 
Invalid `prisma.user.findMany()` invocation:


The table `public.User` does not exist in the current database.
    at Mn.handleRequestError (/Users/chinemerem/Documents/GitHub/sistema/node_modules/@prisma/client/runtime/library.js:121:7753)
    at Mn.handleAndLogRequestError (/Users/chinemerem/Documents/GitHub/sistema/node_modules/@prisma/client/runtime/library.js:121:7061)
    at Mn.request (/Users/chinemerem/Documents/GitHub/sistema/node_modules/@prisma/client/runtime/library.js:121:6745)
    at async l (/Users/chinemerem/Documents/GitHub/sistema/node_modules/@prisma/client/runtime/library.js:130:9633)
    at async handler (webpack-internal:///(api)/./src/pages/api/users.ts:8:19)
    at async K (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:2871)
    at async U.render (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/compiled/next-server/pages-api.runtime.dev.js:21:3955)
    at async DevServer.runApi (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/next-server.js:598:9)
    at async NextNodeServer.handleCatchallRenderRequest (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/next-server.js:269:37)
    at async DevServer.handleRequestImpl (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/base-server.js:810:17)
    at async /Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/dev/next-dev-server.js:339:20
    at async Span.traceAsyncFn (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/trace/trace.js:154:20)
    at async DevServer.handleRequest (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/dev/next-dev-server.js:336:24)
    at async invokeRender (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/lib/router-server.js:173:21)
    at async handleRequest (/Users/chinemerem/Documents/GitHub/sistema/node_modules/next/dist/server/lib/router-server.js:350:24) {
  code: 'P2021',
  clientVersion: '5.20.0',
  meta: { modelName: 'User', table: 'public.User' },
  page: '/api/users'
}
```

Reply:
Yeah makes sense, but I think that error is only when the database isn't seeded? So that would be a larger issue haha